demo/angular/karma.conf.js

```diff
@@ -25,7 +25,7 @@ module.exports = function (config) {
       suppressAll: true // removes the duplicated traces
     },
     coverageReporter: {
-      dir: require('path').join(__dirname, './coverage/rhino-web-angular-demo'),
+      dir: require('path').join(__dirname, './coverage/rhino-angular-demo'),
```

Comment:
you can remove this `karma.conf.js` file as well as `karma` and `jasmine` packages from `package.json`